mail_activity_team/models/mail_activity_team.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class MailActivityTeam(models.Model):
+    _name = "mail.activity.team"
+    _description = 'Mail Activity Team'
+
+    name = fields.Char(string='Name', required=True, translate=True)
+    active = fields.Boolean(string='Active', default=True)
+    res_model_ids = fields.Many2many('ir.model', string='Used models')
```

Comment:
@mreficent can you check how it would be possible to add a domain here that only shows the models that inherit the activity.mixin? Otherwise it is very confusing for the user to see all models here, most of which do not allow to add activities.